Taskfile.yaml

```diff
@@ -11,11 +11,8 @@ env:
 tasks:
   ci:
     desc: Workflow to run in CI
-    deps:
-      - install-gofumpt
-      - install-golangci-lint
     cmds:
-      - task: workspace
```

Comment:
We should keep `- task: workspace` for now since `- task: setup` installs `changie`.
There's a better way to reconcile these two tasks. They serve slightly different purposes but have a lot of overlap